Magitek/Logic/Gunbreaker/Buff.cs

```diff
@@ -1,6 +1,6 @@
 using ff14bot;
-using ff14bot.Managers;
 using Magitek.Extensions;
+using Magitek.Models.Account;
```

Comment:
Is this used? I didn't see anything for it.

Reply:
Yes by BaseSettings (l.36)